docker/prepare.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -xeo pipefail
+
+
+echo "--- unpacking bases for snapcraft/$RISK ---"
+
+# Grab the core snaps (which snapcraft uses as a base) from the stable channel
+# and unpack them in the proper place.
+BASES="core core18 core20 core22"
```

Comment:
it doesn't make much sense to build a snap with a core18 base if the base of the OCI image is 22.04

Reply:
There are many snaps that need `core18`. Why it is wrong to build `core18` with 22.04?